pyth_observer/check/publisher.py

```diff
@@ -46,10 +49,23 @@ def state(self) -> PublisherState:
         return self.__state
 
     def run(self) -> bool:
+        # Skip if not trading
+        if self.__state.status != PythPriceStatus.TRADING:
+            return True
+
+        # Skip if aggregate is not trading
+        if self.__state.aggregate_status != PythPriceStatus.TRADING:
+            return True
+
         # Skip if confidence interval is zero
         if self.__state.confidence_interval == 0:
             return True
 
+        # Pass if publisher slot is far from aggregate slot
+        distance = abs(self.__state.slot - self.__state.aggregate_slot)
+        if distance > 25:
```

Comment:
This value is hard-coded but it is not very important. It is used to filter out the non-trading or behind publishers. We can change it later in the future.